src/migrations/20221810114644-add-suggest-changes-table.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+CREATE TABLE IF NOT EXISTS suggest_change_set (
+  id serial primary key,
+  environment varchar(255) NOT NULL,
+  state varchar(255) NOT NULL,
+  project varchar(255) NOT NULL,
+  created_by varchar(255) NOT NULL,
```

Comment:
Should we use a foreign key for `user` instead if we want to strongly link the change sets to actual user. In future, we are not displaying the creator just as string, but we will need to run queries on it. Having a `created_by: user.username || user.email` will make it error prone and hard to query.

Reply:
makes sense

Reply:
I went through the migrations, almost everywhere `created_by` is set to a text field

Reply:
I guess it depends on the intent of the table, there may not be a lot of other tables where we need to query for drafts per user, where createdat essentially is metadata and the id is the foreign key.